appsso-starter-java/README.md

```diff
@@ -4,4 +4,4 @@ This repository provides an example application used to set up an authentication
 
 ## Usage
 
-[Securing a Workload with AppSSO](https://docs-staging.vmware.com/en/draft/VMware-Tanzu-Application-Platform/1.5/tap/app-sso-app-operators-tutorials-securing-first-workload.html)
+[Securing a Workload with AppSSO](https://docs.vmware.com/en/draft/VMware-Tanzu-Application-Platform/1.5/tap/app-sso-app-operators-tutorials-securing-first-workload.html)
```

Comment:
I don't think we should use non-public links

Reply:
you're right draft is still in here.